vercel.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://openapi.vercel.sh/vercel.json",
+  "outputDirectory": "public",
```

Comment:
Context: This is required because adding `vite` to the dependencies triggered framework detection to run in `static-build` for this repo's Vercel project, which uses a different output directory.

We can't use `"framework": null` here because it doesn't properly override this check. We have to override the `outputDirectory` directly.